tools/cloud_functions/gcs_event_based_ingest/gcs_ocn_bq_ingest/README.md

```diff
@@ -59,7 +59,82 @@ DEFAULT_DESTINATION_REGEX = (
     r")?"                             # [end]yyyy/mm/dd/hh/ group (optional)
     r"(?P<batch>[\w\-_0-9]+)?/"       # batch id (optional)
 )
-`
+```
+
+## Monitoring
+### monitored_tables
+To monitor the status of the tables loaded by the cloud function, you must first add records to the monitored_tables table. The records in this table are used by the gcf_ingest_log and gcf_ingest_latest_by_table views to know which tables to report on.
+
+### gcf_ingest_log
+This gives you a view of all of the attempts to load a given chunk. For example, if we wanted to see the entire history of chunk attempts for a 'table_a', we would run the follwing query:
```

Comment:
chunk is customer specific terminology (and confusing). let's just use the word GCS prefix instead.